src/components/Button/Button.tsx

```diff
@@ -13,63 +13,47 @@ const resetDefault = css`
   background: none;
 `;
 
-const variantSolid = css`
-  ${({ theme }) => css`
-    background-color: ${theme.color.primary};
-    color: ${theme.color.white};
-    &:hover:not(:disabled) {
-      background-color: ${lighten(0.1, `${theme.color.primary}`)};
-      border-color: ${lighten(0.1, `${theme.color.primary}`)};
-    }
-  `}
-`;
-
 const commonStyles = css`
```

Comment:
this button has different animation and color than the button used in header


https://github.com/vazco/forminer/assets/49162755/c89f45f4-5b86-4a42-96b5-7ee5c37db368


https://github.com/vazco/forminer/assets/49162755/e4c501fc-5634-4a78-bbe9-11e8c8fb1e70

Reply:
Thanks, I didn't pay enough attention to this!